string joint_activity
int64 active_joint
bool has_switched
float64 active_joint_normalized